app/models/pusher.rb

```diff
@@ -277,7 +277,7 @@ def notify_unauthorized
       notify("You are not allowed to push this gem.", 403)
     elsif rubygem.unconfirmed_ownership?(owner)
       notify("You do not have permission to push to this gem. " \
-             "Please confirm the ownership by clicking on the confirmation link sent your email #{owner.email}", 403)
+             "Please click the confirmation link we just emailed you to #{owner.email} to confirm you own this gem, so we can verify that you are the rightful owner before allowing pushes.", 403)
```

Comment:
```suggestion
             "Please click the confirmation link we emailed you at #{owner.email} to verify ownership before pushing.", 403)
```

I like the wording you have in the PR description more since it's more clear and direct.